src/state/queries/post-feed.ts

```diff
@@ -43,8 +43,13 @@ type AuthorFilter =
   | 'posts_with_media'
 type FeedUri = string
 type ListUri = string
+/**
+ * Represents a "not found" state, that results in the left-most tab being
+ * selected on the Home screen.
+ */
+export const DEFAULT_FEED_DESCRIPTOR = '__default__'
 export type FeedDescriptor =
-  | 'home'
+  | '__default__'
```

Comment:
This is here just to have a consistent fallback that forces the pager to select the first tab. See other comment further down.

Reply:
`null` is a great idea!